packages/components/src/components/gif.tsx

```diff
@@ -230,18 +230,18 @@ const Gif = ({
         >
             <div style={{ width, height, position: 'relative' }} ref={container}>
                 <picture>
-                    <source type="image/webp" srcSet={rendition.webp} />
+                    <source type="image/webp" srcSet={shouldShowMedia ? rendition.webp : placeholder} />
```

Comment:
the bug fix, check the shouldShowMedia flag, if it's true show it, if it's false show the placeholder